bookshelf.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+class Book:
```

Comment:
should be retreat under "import"
like : 
import pytest


class Book:

Reply:
Thank you, I really didn't see it